.circleci/config.yml

```diff
@@ -243,3 +271,28 @@ workflows:
             - test
             - lint
             - build
+      - release:
+          context:
+            - welcome-ui
+          requires:
+            - test
+            - lint
+            - build
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v\d.\d.\d/
```

Comment:
avec une alpha cela va marcher ?

Reply:
This script is for stable release, for prereleases (that have a different dist-tag on npm), there is another rule here thats includes -alpha and any other keyword: https://github.com/WTTJ/welcome-ui/pull/1893/files/94c59122be14d51d7f3fc74a598a80627d74ef63#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R298